Tests/MapboxMapsTests/Foundation/Integration Tests/FeatureStateIntegrationTests.swift

```diff
@@ -0,0 +1,104 @@
+import Turf
+import XCTest
+@testable import MapboxMaps
+
+internal class FeatureStateIntegrationTests: MapViewIntegrationTestCase {
+
+    internal func testSetFeatureState() {
+        style?.uri = .streets
+        let featureStateExpectation = XCTestExpectation(description: "Wait for feature state  map to be updated.")
+
+        didFinishLoadingStyle = { mapView in
+
```

Comment:
nit: remove new line

Reply:
this is swiftlint doing it's thing i think